exercises/rotational-cipher/examples/success-standard/src/RotationalCipher.hs

```diff
@@ -0,0 +1,9 @@
+module RotationalCipher (rotate) where
+
+import Data.List (elemIndex)
+
+rotate :: Int -> String -> String
+rotate r = map (f r ['a'..'z'] . f r ['A'..'Z'])
+  where f x abc e = case elemIndex e abc of
+                      Nothing  -> e
+                      (Just a) -> abc!!mod (a+x) 26
```

Comment:
huh, I didn't think the parentheses around `Just a` are required, but I don't know